R/render-plot.R

```diff
@@ -112,6 +113,7 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
             func = func,
             width = dims$width,
             height = dims$height,
+            alt = alt,
```

Comment:
Also, above (after `heightWrapper`), define `altWrapper` to ingest `alt` similar to how `height` and `width` are ingested.

```suggestion
            alt = altWrapper(),
```